{"name": "test_get_endpoints_response_time[/]", "status": "failed", "statusDetails": {"message": "Failed: Request to https://rangoljx.github.io/ failed with exception: Response time for https://rangoljx.github.io/ was too long: 11.317197561264038 seconds\nassert 11.317197561264038 < 10", "trace": "endpoint = '/'\n\n    @pytest.mark.parametrize(\"endpoint\", [\n        \"/\",  # 首页\n        \"/about.html\",  # 关于页面\n        \"/blog/\"  # 博客页面\n    ])\n    def test_get_endpoints_response_time(endpoint):\n        \"\"\"\n        测试网站不同端点的GET请求响应时间。\n        参数:\n            - endpoint (str): 要测试的网站端点。\n        测试方法:\n            - 构造完整的URL。\n            - 发送GET请求。\n            - 检查响应时间是否小于设定的阈值。\n        \"\"\"\n        try:\n            response = send_request(\"GET\", endpoint)  # 使用封装的请求函数\n            rtime = response['response_time']\n    \n            # 设定响应时间阈值（例如：1秒）\n            MAX_RESPONSE_TIME = 10\n    \n            # 检查响应时间是否在阈值范围内\n>           assert rtime < MAX_RESPONSE_TIME, f\"Response time for {BASE_URL}{endpoint} was too long: {rtime} seconds\"\nE           AssertionError: Response time for https://rangoljx.github.io/ was too long: 11.317197561264038 seconds\nE           assert 11.317197561264038 < 10\n\ntests\\test_api.py:61: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nendpoint = '/'\n\n    @pytest.mark.parametrize(\"endpoint\", [\n        \"/\",  # 首页\n        \"/about.html\",  # 关于页面\n        \"/blog/\"  # 博客页面\n    ])\n    def test_get_endpoints_response_time(endpoint):\n        \"\"\"\n        测试网站不同端点的GET请求响应时间。\n        参数:\n            - endpoint (str): 要测试的网站端点。\n        测试方法:\n            - 构造完整的URL。\n            - 发送GET请求。\n            - 检查响应时间是否小于设定的阈值。\n        \"\"\"\n        try:\n            response = send_request(\"GET\", endpoint)  # 使用封装的请求函数\n            rtime = response['response_time']\n    \n            # 设定响应时间阈值（例如：1秒）\n            MAX_RESPONSE_TIME = 10\n    \n            # 检查响应时间是否在阈值范围内\n            assert rtime < MAX_RESPONSE_TIME, f\"Response time for {BASE_URL}{endpoint} was too long: {rtime} seconds\"\n        except Exception as e:\n            # 捕获并打印异常，以便调试\n>           pytest.fail(f\"Request to {BASE_URL}{endpoint} failed with exception: {e}\")\nE           Failed: Request to https://rangoljx.github.io/ failed with exception: Response time for https://rangoljx.github.io/ was too long: 11.317197561264038 seconds\nE           assert 11.317197561264038 < 10\n\ntests\\test_api.py:64: Failed"}, "description": "\n    测试网站不同端点的GET请求响应时间。\n    参数:\n        - endpoint (str): 要测试的网站端点。\n    测试方法:\n        - 构造完整的URL。\n        - 发送GET请求。\n        - 检查响应时间是否小于设定的阈值。\n    ", "parameters": [{"name": "endpoint", "value": "'/'"}], "start": 1733994986323, "stop": 1733994997656, "uuid": "9c9f915d-59fd-471c-b46e-71acaadbdacd", "historyId": "de6a0c5986059a770d1feb07c97866c3", "testCaseId": "665eff6bdfbd75f244e4101b5dd24a7f", "fullName": "tests.test_api#test_get_endpoints_response_time", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "LAPTOP-CP2FN3BV"}, {"name": "thread", "value": "33192-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}